go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/smartcontractkit/external-initiator
 go 1.15
 
 require (
+	github.com/Conflux-Chain/go-conflux-sdk v1.0.1
 	github.com/Depado/ginprom v1.2.1-0.20200115153638-53bbba851bd8
 	github.com/FactomProject/basen v0.0.0-20150613233007-fe3947df716e // indirect
 	github.com/avast/retry-go v2.6.0+incompatible
 	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
 	github.com/centrifuge/go-substrate-rpc-client v2.0.0+incompatible
 	github.com/cmars/basen v0.0.0-20150613233007-fe3947df716e // indirect
-	github.com/ethereum/go-ethereum v1.9.24
+	github.com/ethereum/go-ethereum v1.9.25
```

Comment:
Do we need to update dependencies in this PR scope?

Reply:
Our SDK `go-conflux-sdk` use a newer version go-ethereum 1.9.25, so we update it here